pages/common/kubeadm.md

```diff
@@ -0,0 +1,31 @@
+# kubeadm
+
+> Command line interface for creating and managing Kubernetes clusters.
+
+- Create a Kubernetes master node:
+
+`kubeadm init`
+
+- Bootstrap a Kubernetes worker node and join it to a cluster:
+
+`kubeadm join --token {{token}}`
+
+- Create a new bootstrap token with a TTL of 12 hours:
+
+`kubeadm token create --ttl 12h0m0s`
```

Comment:
12h0m0s in tokens (`{{}}`)

Reply:
Should I change the example description, then? To something like "Create a new bootstrap token with a custom TTL"?

Reply:
This has been updated :)